### DISCUSSION TITLE
Pass user_id in ad request?

### DISCUSSION THREAD

**Discussion by Toupoc** [Poing-Studios/godot-admob-editor#50] [Q&A]
Admob has the ability to receive a user_id argument with the ad request. They should then pass this user_id back to your server through serverside verification.  Assuming I already have SSV set up (which I do) how do I pass in the player's user_id in the initial ad request to admob?

**Comment by gumaciel** [comment]
Hi @Toupoc i don't know, how do you know that we receive the `user_id` argument? 

> how do I pass in the player's user_id in the initial ad request to admob?

I think it's not possible
